.gitignore

```diff
@@ -49,3 +49,6 @@ env/
 gamecenter/static/docs
 
 postgres
+
+# ignore the things inside of the log folder
+logs/*log.json
```

Comment:
For when this is actually deployed we may want to put this somewhere more standard, like `/var/log/...`